sample/run_samples.sh

```diff
@@ -1,50 +1,21 @@
-#!/bin/sh
+#!/bin/bash
 
-outdir=$1
+# Exit on error
+set -e
 
-alarabin='../src/alara'
+# ALARA executable
+ALARA=../bin/alara
 
-if ! test -x $alarabin; then
-    alarabin='alara'
-    alarastring=`which $alarabin`
-    if test "X$alarastring" = "X"; then
-	echo "Could not find executable to run sample cases!
-If this is build tree, make sure executable was built successfully.
-If this is the installed sample directory, make sure executable is in path."
-	exit -1
-    else
-	echo "Running sample cases with executable from path $alarastring"
-    fi
-else
-    echo "Running sample cases with executable from local build: $alarabin"
-fi
+# Print version information
+${ALARA} -V
 
-$alarabin -V
+# Setup output directories
+rm -rf   output dump_files
+mkdir -p output dump_files
 
-rm -rf $outdir
-mkdir $outdir
-rm -rf dump_files
-mkdir dump_files
-echo sample1
-$alarabin -v 3 -t $outdir/sample1.tree sample1 > $outdir/sample1.out
-echo sample2
-$alarabin -v 3 -t $outdir/sample2.tree sample2 > $outdir/sample2.out
-echo sample3
-$alarabin -v 3 -t $outdir/sample3.tree sample3 > $outdir/sample3.out
-echo sample4
-$alarabin -v 3 -t $outdir/sample4.tree sample4 > $outdir/sample4.out
-echo sample5
-sed -e "s/\.\/sample5.photonSrc/$outdir\/sample5.photonSrc/" sample5 > sample5.tmp
-$alarabin -v 3 -t $outdir/sample5.tree sample5.tmp > $outdir/sample5.out
-rm sample5.tmp
-echo sample6
-$alarabin -v 3 -t $outdir/sample6.tree sample6 > $outdir/sample6.out
-echo sample7
-$alarabin -v 3 -t $outdir/sample7.tree sample7 > $outdir/sample7.out
-echo sample8
-$alarabin -v 3 -t $outdir/sample8.tree sample8 > $outdir/sample8.out
-echo sample9
-sed -e "s/\.\/sample9.photonSrc_int/$outdir\/sample9.photonSrc_int/" sample9 > sample9.tmp
-sed -i "s/\.\/sample9.photonSrc/$outdir\/sample9.photonSrc/" sample9.tmp
-$alarabin -v 3 -t $outdir/sample9.tree sample9.tmp > $outdir/sample9.out
-rm sample9.tmp
+# Run samples
+samples=sample?
+for sample in ${samples}; do
+  echo ${sample}
+  ${ALARA} -v 2 -t output/${sample}.tree ${sample} > output/${sample}.out
```

Comment:
Testing at verbosity 3 is useful to make sure the tree is being built correctly, but makes diffs harder.  

We could diff the trees instead?

Reply:
Perhaps I can come up with a way to have it skip checking the lines that contain the timing info when doing the diffs

Reply:
I changed it back to verbosity 3, but I also changed the code so that the timing results don't show up until verbosity level 4. Is that acceptable? Another option would be to create a new command line flag that removes all timing results.